src/App.js

```diff
@@ -106,21 +109,17 @@ class App extends Component {
     }
   }
 
-  componentDidMount() {
-    fetch("https://whateverly-datasets.herokuapp.com/api/v1/nationalParks1810")
-      .then(data => data.json())
-      .then(results => {
-        results.nationalParks1810.forEach(park => {
-          if (park.parkName === 'Sequoia') {
-            park.urlCode = 'seki2';
-          }
-        });
-        this.setState({
-          parks: results.nationalParks1810,
-          currentParksToShow: results.nationalParks1810
-        });
+  async componentDidMount() {
```

Comment:
This is great! This is awesome we have the park data from the national park service now